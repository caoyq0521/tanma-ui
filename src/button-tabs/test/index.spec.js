// import { mount } from '@vue/test-utils';

// describe('tm-button-tabs', () => {
  
// })
  