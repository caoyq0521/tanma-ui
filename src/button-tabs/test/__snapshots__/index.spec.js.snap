// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tm-button-tabs render button-tabs 1`] = `
<div class="tm-button-tabs">
  <div class="tm-button-tabs__step">
    <div class="tm-button-tabs__step-prve">
      上一步
    </div>
    <div class="tm-button-tabs__step-next">
      下一步
    </div>
  </div>
  <div class="tm-button-tabs__module"></div>
  <div class="tm-button-tabs__date"></div>
  <div class="tm-button-tabs__certainty">
    <div class="tm-button-tabs__certainty-cancel">取消</div>
    <div class="tm-button-tabs__certainty-confirm">确定</div>
  </div>
  <div class="tm-button-tabs__save"></div>
</div>
`;
